x-pack/plugins/secops/public/components/page/network/kpi_network/translations.ts

```diff
@@ -17,3 +17,11 @@ export const UNIQUE_ID = i18n.translate('xpack.secops.kpiNetwork.source.uniquiId
 export const ACTIVE_AGENTS = i18n.translate('xpack.secops.kpiNetwork.source.activeAgentsTitle', {
   defaultMessage: 'Active Agents',
 });
+
+export const UNIQUE_PRIVATE_IP = i18n.translate('xpack.secops.kpiNetwork.source.uniquePrivateIps', {
```

Comment:
I think per the guidelines of `i18n` we have you want to end this with `Title` as in:

```ts
i18n.translate('xpack.secops.kpiNetwork.source.uniquePrivateIpsTitle', {
```